script.js

```diff
@@ -0,0 +1,77 @@
+class BookLibrary {
+  constructor() {
+    this.myBooks = [];
+    this.addBtn = document.getElementById('add-btn');
+    this.listBooks = document.getElementById('list-books');
+
+    const storedBooks = localStorage.getItem('myBooks');
+    if (storedBooks) {
+      this.myBooks.push(...JSON.parse(storedBooks));
+    }
+
+    this.addBtn.addEventListener('click', this.addBook.bind(this));
+
+    // Initial display of books
+    this.showBooks();
+  }
+
+  showBooks() {
+    this.listBooks.innerHTML = '';
+
+    this.myBooks.forEach((myBook) => {
+      const myBookDiv = document.createElement('div');
+      myBookDiv.classList.add('row');
+
+      const bookDetails = document.createElement('h4');
+      bookDetails.textContent = `${myBook.bookTitle} by ${myBook.bookAuthor}`;
+
+      const removeBtn = document.createElement('button');
+      removeBtn.classList.add('remove-btn');
+      removeBtn.textContent = 'Remove';
+      removeBtn.addEventListener('click', () => {
+        this.removeBook(myBook.bookTitle);
+      });
+
+      myBookDiv.appendChild(bookDetails);
+      myBookDiv.appendChild(removeBtn);
+      this.listBooks.appendChild(myBookDiv);
+    });
+  }
+
+  addBook() {
+    const bookTitle = document.getElementById('title').value.trim();
+    const bookAuthor = document.getElementById('author').value.trim();
+    const errorMsg = 'Both Title and Author must be filled out!';
+
+    if (bookTitle === '' || bookAuthor === '') {
+      // eslint-disable-next-line no-alert
+      alert(errorMsg);
+      return;
+    }
+
+    this.myBooks.push({ bookTitle, bookAuthor });
+    localStorage.setItem('myBooks', JSON.stringify(this.myBooks));
+
+    // Call showBooks() to see all books
+    this.showBooks();
+
+    // Clear the input fields values
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+  }
+
+  removeBook(bookTitleToRemove) {
+    const myBooksUpdated = this.myBooks.filter((myBook) => myBook.bookTitle !== bookTitleToRemove);
+    localStorage.setItem('myBooks', JSON.stringify(myBooksUpdated));
```

Comment:
- [ ] Good job creating `removeBook` methods, however, when books have the same title, then when we remove one, all of them will be removed 😭. I'll suggest you use an unic `id` inside of using `booTitle` 👍.

eg: When you are adding a book, from there we can generate olso the `id`. you can use another approach, but let's take an example from this:

````js
 addBook() {
       const id =  this.myBooks.length + 1;
       .
       .
       .
}
```` 
Then update your `removeBook` method. Kindly, fix it 💪